catalog-rest-service/src/main/resources/ui/src/interface/types.d.ts

```diff
@@ -322,4 +323,51 @@ declare module 'Models' {
     columns: Array<string>;
     rows: Array<Array<string>>;
   };
+
+  // topic interface start
+  export interface Topic {
```

Comment:
No need to add this interface in types.d.ts, use it directly from generated types

Reply:
okay.